### DISCUSSION TITLE
Subscription Duration Options for User Accounts

### DISCUSSION THREAD

**Discussion by amirprokh98** [DefGuard/defguard#842] [Ideas]
I suggest implementing subscription duration options for creating user accounts, such as monthly, yearly, and potentially other time frames. This flexibility would allow users to choose a plan that best fits their needs, making it easier for manage user accounnts
Thank you for considering this suggestion. I believe it could greatly benefit overall success of your application.

Question:  How many users and locations can be added without a license?

**Comment by teon** [comment]
We will rollout yearly plans this week, thank you for your suggestion.

Regarding your question: How many users and locations can be added without a license?

The Open Source version does not have any limits to users or locations, it just doesn't include the extra enterprise features: https://docs.defguard.net/enterprise/all-enteprise-features

Hope you like defguard!

**Comment by amirprokh98** [comment]
Sorry, I mistakenly conveyed my point.
I wasn’t talking about the annual license; I meant to suggest that an option be added to the defguard admin panel when creating user to specify how long the user account should be active, after which it would be disabled account.
